easybuild/easyblocks/n/numpy.py

```diff
@@ -166,6 +166,13 @@ def get_libs_for_mkl(varname):
         if fft:
             extrasiteconfig += "\n[fftw]\nlibraries = %s" % fft
 
+        if get_software_root('SUITESPARSE'):
+            suitesparseroot = get_software_root('SUITESPARSE')
```

Comment:
move this above the `if` and use `if suitesparseroot:`

also, please use the 'official' name `SuiteSparse` (`get_software_root` will convert it to uppercas to check `$EBROOTSUITESPARSE`)